<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-MJ1T9AR" ignored="0" name="Test" tests="8" failures="1" timestamp="2022-04-17T04:19:20 EET" time="1.636" errors="0">
  <testcase name="Validate_invaild_reponse_code_TCs" time="1.002" classname="QA.TestAPI.Product"/>
  <testcase name="Validate_invalid_data_TCs" time="0.028" classname="QA.TestAPI.Product"/>
  <testcase name="Validate_reponse_data_TCs" time="0.315" classname="QA.TestAPI.Product"/>
  <testcase name="Validate_vaild_reponse_code_TCs" time="0.012" classname="QA.TestAPI.Product"/>
  <testcase name="Validate_invaild_reponse_code_TCs" time="0.051" classname="QA.TestAPI.SearchAboutProduct">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &amp;lt;500&amp;gt; but was &amp;lt;200&amp;gt;.
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <500> but was <200>.

at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:237)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:249)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:493)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:674)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:190)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:58)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:168)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:126)
at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:134)
at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
at io.restassured.internal.ValidatableResponseImpl.super$2$statusCode(ValidatableResponseImpl.groovy)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1217)
at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.invokeMethodOnSuperN(ScriptBytecodeAdapter.java:144)
at io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy:142)
at io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy)
at QA.TestAPI.SearchAboutProduct.Validate_invaild_reponse_code_TCs(SearchAboutProduct.java:19)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 42 stack frames]]>
    </failure>
  </testcase> <!-- Validate_invaild_reponse_code_TCs -->
  <testcase name="Validate_invalid_data_TCs" time="0.047" classname="QA.TestAPI.SearchAboutProduct"/>
  <testcase name="Validate_reponse_data_TCs" time="0.081" classname="QA.TestAPI.SearchAboutProduct"/>
  <testcase name="Validate_vaild_reponse_code_TCs" time="0.067" classname="QA.TestAPI.SearchAboutProduct"/>
</testsuite> <!-- Test -->
